### ISSUE TITLE
Updated to 2.0.4 could not find type definitions

### ISSUE THREAD

**Issue by dupkey** [MariaDB/mariadb-connector-nodejs#49]
I'm not able to find TypeScript definitions included with the NPM release.

Works if installed from the github repo.

**Comment by knoxcard** [comment]
or you can use yarn

```
yarn add https://github.com/MariaDB/mariadb-connector-nodejs.git
```

**Comment by dupkey** [comment]
Yup, that's how I installed it.

**Comment by rusher** [comment]
reproduced. 
The strange thing is that npm package doesn't have 'types/index.d.ts', that's what missing. 
Package.json indicate to export this file : 
```
 "files": [
    "lib",
    "types/index.d.js",
    "promise.js",
    "callback.js"
  ],
```
 I must have missed something. 
_re-reading package.json definition_

**Comment by rusher** [comment]
ohh, "types/index.d.**js**", not "types/index.d.**ts**"
This will need a new release

**Comment by rusher** [comment]
corrected with new [2.0.5 ](https://github.com/MariaDB/mariadb-connector-nodejs/releases/tag/2.0.5) release
